cpp/src/io/csv/writer_impl.cu

```diff
@@ -367,10 +374,10 @@ void write_chunked(data_sink* out_sink,
 
   CUDF_EXPECTS(str_column_view.size() > 0, "Unexpected empty strings column.");
 
-  cudf::string_scalar newline{options.get_line_terminator()};
+  cudf::string_scalar newline(options.get_line_terminator(), true, stream);
```

Comment:
```suggestion
  cudf::string_scalar newline{options.get_line_terminator(), true, stream};
```
nit: curly brackets are preferred for initialization.
https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es23-prefer-the--initializer-syntax

The same in other places.

Reply:
Thanks, addressed!